docs/recipes/client-server/fable-remoting.md

```diff
@@ -59,6 +59,13 @@ let myApi =
     |> Remoting.buildProxy<IMyApi>
 ```
 
+> Note: We have made your life easier by creating Metapackages which abstract some of this boilerplate. You can also create the client proxy like so:
```

Comment:
I would remove the above sample - there's no need for both. Or at the very least, put the preferred way (`Api.makeProxy`) first.